utils/firestore.js

```diff
@@ -2014,6 +2013,38 @@ const addKitAssemblyData = async (data) => {
     }
 }
 
+const updateKitAssemblyData = async (data) => {
+    try {
+        const snapShot = await db.collection('kitAssembly').where('687158491', '==', data['687158491']).get();
+
+        if (snapShot.empty) return false
+        const docId = snapShot.docs[0].id;
+ 
+        await db.collection('kitAssembly').doc(docId).update({
+            '194252513': data['194252513'],
+            '259846815': data['259846815'],
+            '531858099': data['531858099'],
+            '690210658': data['690210658'],
+            '786397882': data['786397882']
+        })
+        return true;
+    }
+    catch(error){
+        console.error(error);
+        return new Error(error);
+    }
+}
+
+const checkCollectionUniqueness = async (id) => {
+    try {
+        const snapShot = await db.collection('kitAssembly').where('259846815', '==', id).get();
+        if (snapShot.docs.length === 0) { return true }
```

Comment:
does `snapShot.docs.length === 0` do the same thing as `snapShot.empty` on line 2020? If so, I'd just suggest picking one and using it in both places for consistency